data/json/items/armor/boots.json

```diff
@@ -930,6 +930,28 @@
     "flags": [ "VARSIZE", "WATERPROOF" ],
     "armor": [ { "encumbrance": 20, "coverage": 65, "covers": [ "foot_l", "foot_r", "leg_r", "leg_l" ] } ]
   },
+  {
+    "id": "motor_police_boots",
+    "repairs_like": "boots_larmor",
+    "type": "ARMOR",
+    "name": { "str": "pair of motorcycle police boots", "str_pl": "pairs of motorcycle police boots" },
+    "description": "Custom made knee-high leather boots issued to motor officers.  Warm, durable and quite tough.",
+    "weight": "1520 g",
+    "volume": "3750 ml",
+    "price": 8000,
+    "price_postapoc": 1250,
+    "to_hit": -1,
+    "bashing": 1,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "looks_like": "knee_high_boots",
+    "color": "dark_gray",
+    "armor": [ { "covers": [ "foot_l", "foot_r", "leg_r", "leg_l" ], "coverage": 65, "encumbrance": 15 } ],
+    "warmth": 20,
+    "material_thickness": 2,
+    "environmental_protection": 2,
+    "flags": [ "VARSIZE", "WATERPROOF", "OUTER", "STURDY" ]
```

Comment:
What purpose does `"OUTER"` serve here on the shoes?  Does it allow other shoes to be worn under it?

Reply:
"OUTER" lets you wear the breeches with the boots without adding additional encumbrance. It seems like a bit of a dirty fix to me but without it it would be a bit unrealistic. And no. It still forbids you from wearing any other footwear with it while I played.